### PR TITLE
Pauline/feat/3 condition1

### LINKED ISSUE #3: Condition 1 method

### LINKED ISSUE #42: Wrong angle used to compare in smallestCircleRadius
Angles should be compared to PI/2 and not PI to check whether they are obtuse.

### PR DESCRIPTION
**Pull request by paulinegarelli**

### CODE REVIEW

File: decide/src/main/java/org/example/Decide.java [hunk 1]
```diff
@@ -26,7 +26,10 @@ public boolean condition0() {
     }
 
     public boolean condition1() {
-        return true;
+        return IntStream.range(0, settings.NUMPOINTS - 2).anyMatch(
```
Comment (hampfh) [inline decide/src/main/java/org/example/Decide.java:51]: 👍

File: decide/src/main/java/org/example/Point.java [hunk 2]
```diff
@@ -16,4 +16,45 @@ public double distance(Point other) {
...
+        return Math.acos(scalarProduct / (distance(p2) * distance(p3)));
+    }
+
+    /**
+     * Computes the radius of the smallest circle containing all three points (p1, p2, p3)
+     * If one angle of the triangle is obtuse:
+     * the smallest circle containing the triangle has the opposite side as diameter
+     * Else:
+     * the smallest circle is the circumscribed circle.
+     * the radius is computed thanks to the law of sines : R = a * b * c / (4 * S)
```
Comment (hampfh) [inline decide/src/main/java/org/example/Point.java]: From this comment it looks like the formula belongs to the law of sines, but from what I can tell this is the formula for a circumscribed circle. Might be worth just clarifying this

File: decide/src/main/java/org/example/Point.java [hunk 3]
```diff
@@ -16,4 +16,45 @@ public double distance(Point other) {
...
+     * @param p1 first point
+     * @param p2 second point
+     * @param p3 third point
+     * @return radius of the smallest circle containing p1, p2 and p3
+     */
+    public static double smallestCircleRadius(Point p1, Point p2, Point p3) {
+        // checking if any of the angles is obtuse
+        if (p1.angle(p2, p3) > Math.PI) {
```
Comment (hampfh) [inline decide/src/main/java/org/example/Point.java]: If you want to check if the angle is obtuse shouldn't you check `> Math.PI / 2`?

**Review by hampfh — CHANGES_REQUESTED** [review]
Nice work overall, I have one question regarding a comment and one thing that I think is a bug (please correct me if I'm wrong)

File: decide/src/main/java/org/example/Point.java (hunk 2 above)
Comment (paulinegarelli) [reply decide/src/main/java/org/example/Point.java]: Yes, what I meant is that we use the law of sines to compute the radius of the circumscribed circle but you're right I'll change that

File: decide/src/main/java/org/example/Point.java (hunk 3 above)
Comment (paulinegarelli) [reply decide/src/main/java/org/example/Point.java]: Right, thanks!

**Review by hampfh — APPROVED** [review]
Thank you for making the changes, great work!

**Pull request merged**